### PR TITLE
WT-12284 Add support for verify operation in workgen

### PR DESCRIPTION
**Pull request by mm-ng**

### CODE REVIEW

**Comment by mm-ng** [comment]
I was originally planning to add some workloads for pre-fetch but I think that can be done in another ticket. This one is just to add verify into workgen and can be used more generally when testing performance.

**Review by korteland — COMMENTED** [review]
The stuff that's been added looks fine as far as the pattern-matching goes, but it's harder for me to find spots where something needed to change but didn't. I'm choosing to trust that you covered all the bases here, I can check that as well if you want.

File: bench/workgen/workgen.cpp [hunk 1]
```diff
@@ -2385,6 +2402,8 @@ Operation::get_static_counts(Stats &stats, int multiplier)
         stats.checkpoint.ops += multiplier;
     else if (_optype == OP_RTS)
         stats.rts.ops += multiplier;
+    else if (_optype == OP_VERIFY)
+        stats.verify.ops += multiplier;
```
[the remark this answers is not in the document]
Comment (mm-ng) [reply bench/workgen/workgen.cpp:2406]: @korteland Yes, I agree. I tried to follow the logic that was already there. I'm not sure if I'm correct - it looks like the multiplier is usually 1, but I think sometimes it's used if we want to repeat groups of operations.

[the remark this answers is not in the document]
Comment (korteland) [reply bench/workgen/workgen.cpp:2406]: Cool, thanks.

File: bench/workgen/workgen.cpp [hunk 2]
```diff
@@ -2234,6 +2237,14 @@ Operation::Operation(OpType optype, const std::string &config)
     init_internal(nullptr);
 }
 
+Operation::Operation(OpType optype, Table table, std::string config)
```
Comment (etienneptl) [inline bench/workgen/workgen.cpp:2240]:
```suggestion
Operation::Operation(OpType optype, Table table, const std::string &config)
```

File: bench/workgen/workgen.h [hunk 3]
```diff
@@ -313,6 +314,8 @@ struct Operation {
     Operation();
     Operation(OpType optype, Table table, Key key, Value value);
     Operation(OpType optype, Table table, Key key);
+    // Constructor with string and table applies to VERIFY
+    Operation(OpType optype, Table table, std::string config);
```
Comment (etienneptl) [inline bench/workgen/workgen.h:318]:
```suggestion
    Operation(OpType optype, Table table, const std::string &config);
```

**Review by etienneptl — COMMENTED** [review]
Looks like a good improvement! Thank you for doing this, I left a few comments.

**Review by korteland — APPROVED** [review]
LGTM!

**Pull request merged**